src/fsharp/FSharp.Core/map.fsi

```diff
@@ -22,6 +22,12 @@ namespace Microsoft.FSharp.Collections
         /// <returns>The resulting map.</returns>
         member Add: key:'Key * value:'Value -> Map<'Key,'Value>
 
+        /// <summary>Returns a new map with the value stored under key changed according to f.</summary>
+        /// <param name="key">The input key.</param>
+        /// <param name="f">The change function.</param>
+        /// <returns>The resulting map.</returns>
+        member Change: key:'Key * f:('Value option -> 'Value option) -> Map<'Key,'Value>
```

Comment:
This should also have the experimental attribute

Reply:
@cartermp , github has the ability to hide whitespace changes for a pr.

![image](https://user-images.githubusercontent.com/5175830/84190852-6cee1480-aa4c-11ea-9ab9-3e7041e29279.png)